other/install_2-1_mysql.sql

```diff
@@ -320,8 +321,8 @@ CREATE TABLE {$db_prefix}log_comments (
 #
 
 CREATE TABLE {$db_prefix}log_digest (
-  id_topic MEDIUMINT UNSIGNED NOT NULL,
-  id_msg INT(10) UNSIGNED NOT NULL,
+  id_topic MEDIUMINT UNSIGNED NOT NULL DEFAULT '0',
```

Comment:
default value should be empty,
other wise you can insert data which is not valid.

Reply:
Are you sure you want these removed?   They were inserted to bring it consistent with similar columns elsewhere thoughout the DB...

Reply:
We touched base & agreed we'll leave these as-is.